### ISSUE TITLE
Compare Nerdist and Geek & Sundry feeds

### ISSUE THREAD

**Issue by jpgill86** [critrolesync/critrolesync.github.io#110]
PR #109 showed that, in a recent transition from Nerdist to Geek & Sundry controlling the old podcast feed, many things changed:
- Some duration changes may require re-syncing timestamps.
- New Spotify / Google Podcast URLs should be used for every episode.
- It's possible that the files were re-encoded in a new format that gets rid of the CBR vs. ABR issue. This should be investigated.

**Comment by jpgill86** [comment]
A couple days after what I called a "transition" from Nerdist to Geek & Sundry, where the Nerdist feed became inaccessible and I discovered a G&S feed was available, the Nerdist feed came back.

After doing some digging on the Internet Archive WayBack Machine (ff4589c), I discovered that the G&S feed has been available for over a year. Additionally, critrole.com has linked to it all this time (https://critrole.com/campaign-1-podcast), suggesting it may be endorsed as "official". Furthermore, I noticed that the podcast app I use on my phone shows I am now subscribed to the G&S feed, when I am pretty sure I originally subscribed to the Nerdist feed, suggesting a 301 redirect may have been implemented at some point (although that seems to no longer be the case since I was able to re-subscribe to the Nerdist feed).

These facts recommend the G&S feed, but other facts recommend against it. Just a couple days later, it started listing some episodes as having zero duration (#111). This is problematic because the duration published in the feed is used to determine timestamps (https://github.com/critrolesync/critrolesync.github.io/issues/93#issuecomment-1474303044). Additionally, the G&S audio files seem to differ a little from the Nerdist files (see #109). For reasons I still don't completely understand, autosync was producing inexact timestamps for a G&S audio file that I tested (C1E44), despite working well for the Nerdist version.

I have decided, for now at least, to continue using the Nerdist feed for syncing. CritRoleSync will continue to link to the Nerdist Spotify podcast for consistency. This has the drawback of time conversions possibly not being exact for someone subscribed to the G&S feed, but I hope these discrepancies will not be more than ~12 seconds or so, as seen in #109.
